cli/Cargo.toml

```diff
@@ -53,7 +53,7 @@ browser = [
 	"wasm-bindgen",
 	"wasm-bindgen-futures",
 	"browser-utils",
-	"service",
+	"sc-service",
```

Comment:
@bkchr sc-service looks to bring in `sc-client-db`, which then brings in `parity-db`, which uses `memmap` which is incompatible with the web-wasm target...